cmd/kubeadm/app/cmd/phases/init/preflight.go

```diff
@@ -75,6 +75,10 @@ func runPreflight(c workflow.RunData) error {
 		return nil
 	}
 
+	if err := preflight.RunImagesExistCheck(utilsexec.New(), data.Cfg(), data.IgnorePreflightErrors()); err == nil {
```

Comment:
this type of calls ignore early errors from the CRI client.

Reply:
I think it was assuming that they would be repeated below.